src/main/java/com/libraryman_api/member/MemberService.java

```diff
@@ -46,9 +48,14 @@ public MemberService(MemberRepository memberRepository, NotificationService noti
      *
      * @param pageable the pagination information, including the page number and size
      * @return a {@link Page} of {@link Members} representing all members
+     * @throws InvalidSortFieldException if an invalid sortBy field is specified
      */
     public Page<Members> getAllMembers(Pageable pageable) {
-        return memberRepository.findAll(pageable);
+        try {
```

Comment:
What sorting parameters can be passed here?

Reply:
For **Members**, any field in the `Members` model (e.g., name, email, etc.) can be used for sorting.